Makefile

```diff
@@ -54,6 +55,10 @@ clean-docs:
 	make -C doc clean
 	rm -rf doc/code/api
 
+.PHONY : format
+format: 
+	black -l 100 --check --diff strawberryfields
```

Comment:
What does the --check and --diff do here?

Reply:
I believe `check` would simply output which files would be updated, without actually writing to the files (`diff` might be similar, just printing the diff as well; not sure if it writes or not though).

Should probably not need either here if we actually wish to have the command update the files.

Reply:
Yep, they show files to change and a diff rather than autoformatting. Happy to remove them.